packages/amplify-graphql-auth-transformer/src/__tests__/non-model-auth.test.ts

```diff
@@ -173,4 +182,68 @@ describe('@auth directive without @model', () => {
     expectNoDirectives(queryType!);
     expectOneDirective(getField(queryType, 'getSecret'), 'aws_iam');
   });
+
```

Comment:
While you're in here, we should add a test case for SQL support for field-level auth on non-model types.

Reply:
I will add in a follow-up PR.